### DISCUSSION TITLE
Deno support?

### DISCUSSION THREAD

**Discussion by Nezteb** [adonisjs/core#4191] [Ideas]
I know AdonisJS is "a fully featured web framework for Node.js" and Deno is not Node.js, but given its popularity I think it'd be worth considering Deno support. 😄 

As far as I can tell, AdonishJS doesn't use ESM, so it might be a lot of work: https://dev.to/craigmorten/guide-to-porting-node-modules-to-deno-41cd

I also created a Deno module request for AdonisJS here: https://github.com/denoland/wanted_modules/issues/82

**Comment by Julien-R44** [comment]
unfortunately, we're not interested in supporting multiple runtimes. you can read this to know more : https://github.com/thetutlage/meta/discussions/7

but feel free to try to make AdonisJS compatible with Deno. It would be cool to see !

**Comment by jalexmelendez** [comment]
Deno 2.0 is out now, It allows using node packages

**Reply by SergiuSavva** [reply]
Has the compatibility been tested?

**Reply by hbatalhaStch** [reply]
cc @Julien-R44
